assets/css/admin.scss

```diff
@@ -268,6 +273,133 @@ h2 .everest-forms-help-tip {
 	}
 }
 
+/**
+ * Form Builder styles
+ */
+.everest-forms_page_edit-evf-form {
```

Comment:
class name is declared using dash(-) why underscore(_) is used.

Reply:
@bnodsnr because of wordpress `get_current_screen()` thows this in body class.
![capture](https://user-images.githubusercontent.com/3774827/41036320-63d2c72c-69af-11e8-992d-b579da04b260.JPG)